src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationLiveTests.cs

```diff
@@ -622,6 +642,41 @@ public void ConfigurationSettingEquals()
             testSettingDiffTags.Tags.Add("tag3", "test_value3");
             Assert.AreNotEqual(s_testSetting, testSettingDiffTags);
         }
+
+        private bool SettingSelectoComparissonr(SettingSelector actual, SettingSelector other)
+        {
+            if (actual != null && other == null) return false;
+            if (actual.Keys.Except(other.Keys).ToList().Count != 0) return false;
+            if (actual.Keys.Except(other.Keys).ToList().Count != 0) return false;
+            if (!actual.Fields.Equals(other.Fields)) return false;
+            if (actual.AsOf != other.AsOf) return false;
+            
+            return true;
+        }
+
+        [Test]
+        public void SettingSelectorCloneWithBatchLink()
+        {
+            SettingSelector selector = new SettingSelector()
+            {
+                Keys = new List<string> { "key1", "key2", "key3" },
+                Labels = { "label1" },
+                Fields = SettingFields.Key | SettingFields.Label | SettingFields.Value,
+                AsOf = DateTimeOffset.Now
+            };
+
+            Type type = typeof(SettingSelector);
+            MethodInfo cloneMethod = type.GetMethods(BindingFlags.NonPublic | BindingFlags.Instance).Where(x => x.Name == "CloneWithBatchLink").First();
```

Comment:
Add `InternalsVisibleToAttribute`. See https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Azure.Base/data-plane/Azure.Base/Properties/AssemblyInfo.cs#L4